firedrake/slope_limiter/vertex_based_limiter.py

```diff
@@ -86,11 +81,9 @@ def compute_bounds(self, field):
         self.max_field.assign(-1.0e10)  # small number
         self.min_field.assign(1.0e10)  # big number
 
-        par_loop(self._min_max_loop,
-                 dx,
-                 {"maxq": (self.max_field, RW),
-                  "minq": (self.min_field, RW),
-                  "q": (self.centroids, READ)})
+        # FIXME : Relies on implementation detail in PyOp2
```

Comment:
You should elaborate this "implementation detail" thing a little.

Spelling: PyOP2, all Ps are capitcal. There should be no space before colon.